bifrost-fastify/index.ts

```diff
@@ -178,7 +185,7 @@ export const viteProxyPlugin: FastifyPluginAsync<
         }
 
         const { layout, layoutProps } = getLayout(reply);
-        if (!layout) {
+        if (!isPageContext && !layout) {
```

Comment:
if we did not receive a layout from rails, we passthru. However, we cannot fully passthru on a pageContext request. We still need to at least wrap the proxied content in the pageContext json object.
spec https://github.com/Alignable/bifrost/blob/1eb3ffdf7e02e911f9629f5b735eb05b65b15dd6/tests/e2e/specs/e2e.spec.ts#L178